examples/tutorials/analysis-3d/flux_profiles.py

```diff
@@ -107,11 +107,11 @@
 #
 
 regions = make_orthogonal_rectangle_sky_regions(
-    start_pos=SkyCoord("10d", "0d", frame="galactic"),
-    end_pos=SkyCoord("350d", "0d", frame="galactic"),
+    start_pos=SkyCoord("9d", "0d", frame="galactic"),
+    end_pos=SkyCoord("351d", "0d", frame="galactic"),
```

Comment:
The `FluxProfileEstimator` should not use `to_spectrum_dataset` but update the `mask_fit` the could be done with a context manager as in #5387 but more general such as it that takes a mask in input not only an energy range.

Reply:
Really? 
But if you work with a 3D dataset, then you need a specific model for the flux you try to determine. So you could use a spatial model template following the region and fit it. If you do a joint fit then you could even determine the leakage from one region to the other. 
This could be a valid approach but this seems a bit different in terms of use case.